crates/cmd/src/report.rs

```diff
@@ -0,0 +1,206 @@
+use clap::ValueEnum;
+use std::path::PathBuf;
+use std::process::{ExitCode, Termination};
+use wardstone::key;
+use wardstone::primitive::asymmetric::Asymmetric;
+use wardstone_core::primitive::hash::Hash;
+
+use crate::Verbosity;
+
+const INDENT: &str = "         ";
+
+pub(crate) enum GoodPath {
```

Comment:
I find the naming a bit confusing. It feels like it implies what is a valid path rather than signify a successful audit. So perhaps we could do `Success` here?

Reply:
Yeah, I agree. Naming is hard. I personally find `Success` and `Failure` a bit too generic. How do you feel about `PassedAudit`, `FailedAudit`?

Reply:
That also works.